src/Microsoft.AspNetCore.Authentication/IClaimsTransformer.cs

```diff
@@ -14,8 +15,8 @@ public interface IClaimsTransformer
         /// <summary>
         /// Provides a central transformation point to change the specified principal.
         /// </summary>
-        /// <param name="principal">The principal to transform.</param>
+        /// <param name="context"><see cref="ClaimsTransformationContext"/> containing principal to transform and current <see cref="HttpContext"/>.</param>
```

Comment:
Not sure if `<see cref="HttpContext"/>` should be here. For VS to find it here adding `using Microsoft.AspNetCore.Http;` is needed and when getting IntelliSense it adds `Http.HttpContext` anyways:

![image](https://cloud.githubusercontent.com/assets/7989797/13551496/afa99a70-e344-11e5-989c-ce577f838676.png)


Maybe it should be written in plain text?